File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int ans = 0;
        for(int i=0;i<sentences.length;i++){
            int count = 1;
            for(int j=0;j<sentences[i].length();j++){
                if(sentences[i].charAt(j) == ' '){
                    count++;
                }
            }
            ans = Math.max(count,ans);
        }
        return ans;
    }
}

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    let max = 0;
    for(let i = 0;i<sentences.length;i++){
        let count = 1;
        for(let j = 0;j<sentences[i].length;j++){
            if(sentences[i].charAt(j) == ' '){
                count++;
            }
        }
        max = Math.max(max,count);
    }
    return max;
};